### ISSUE TITLE
Support stdlib (`use.std::math::u64` etc) in miden repl

### ISSUE THREAD

**Issue by bergkvist** [0xPolygonMiden/miden-vm#930]
Related:
- https://github.com/0xPolygonMiden/miden-vm/issues/637

This probably doesn't work because miden repl commands are placed between begin and end, and you'd need to do the import on the top-level.

One option would be to allow the miden repl command to take in a "include"/"top-level"/"definitions"-file that is executed on the top-level before begin/end (and which itself doesn't contain begin/end), but only imports and custom procedures.

Example of how it could work (includes.masm):
```
use.std::math::u64

proc.u64checked_add3
    exec.u64::checked_add
    exec.u64::checked_add
end
```

Then, we can execute the repl like so:
```
$ miden repl --includes=includes.masm

========================== Miden REPL ============================

Available commands:

!stack: displays the complete state of the stack
!mem: displays the state of the entire memory
!mem[i]: displays the state of the memory at address i
!undo: remove the last instruction
!program: display the program
!help: prints out all the available commands

>> push.0xffffffff.0x00000000
0 4294967295 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
>> push.0xffffffff.0x00000000
0 4294967295 0 4294967295 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
>> push.0xffffffff.0x00000000
0 4294967295 0 4294967295 0 4294967295 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
>> exec.u64checked_add3
2 4294967293 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
```

If the includes-file contains begin/end, this part could be ignored - meaning it becomes easier to play around with existing miden programs in the repl.

**Comment by bobbinth** [comment]
This could work, but I wonder if for imports specifically we could just add another command to REPL. Maybe something like `!use`. So, to import a module we'd be able to type something like:

```
>> !use std::math::u64
```

We would need to add a command line argument to REPL to make sure it starts up with the right libraries loaded. This could be done via a path to the library file to be loaded (though, for `stdlib` could could be a special flag). For example:
```
$ miden repl --use-stdlib
```

**Comment by Overcastan** [comment]
Closed by #1162.
